.drone.yml

```diff
@@ -0,0 +1,38 @@
+pipeline:
```

Comment:
@ashb should we add a notify step at the end? Should have examples in the other repos to post to slack.

Reply:
Good  call.